examples/pytorch/token-healing/README.md

```diff
@@ -0,0 +1,44 @@
+<!-- back to top link -->
```

Comment:
`examples/research/token-healing` would be a better place to hold this example :) The examples in `examples/pytorch` are selected by the `transformers` team and actively maintained 🤗